src/transformers/modeling_albert.py

```diff
@@ -651,6 +653,14 @@ def forward(
         prediction_scores, sop_scores = outputs[:2]
 
         """
+        
+        if 'masked_lm_labels' in kwargs:
+            warnings.warn(
+                "The `masked_lm_labels` argument is deprecated and will be removed in a future version, use `labels` instead.",
+                DeprecationWarning
+            )
+            labels = kwargs.pop("masked_lm_labels")
```

Comment:
You probably want a `pop("masked_lm_labels", None)` here to not throw if `masked_lm_labels` is None.

Reply:
It can't hurt, but I don't see in which case it would throw since we are inside `"masked_lm_labels" in kwargs`. If the kwargs is `{"masked_lm_labels": None}`, it doesn't throw.

Reply:
Oops yes indeed, read too fast